spec/factories/character_factory.rb

```diff
@@ -4,7 +4,7 @@
   factory :character do
     name { Faker::Games::WorldOfWarcraft.unique.hero }
     species { "Humanoid (#{Faker::Games::DnD.species})" }
-    level { (1..20).to_a.sample }
+    level { Faker::Number.between(from: 1, to: 19).to_i+1 }
```

Comment:
@DanielEFrampton can we add this as "allowed" in hound since rubocop wants these spaces?

Reply:
We can make Hound do whatever we want, so yep; are you saying that your local Rubocop installation is giving you different feedback than Hound? There should be a way to tell your local Rubocop to use our .rubocop.yml so that you're getting the same feedback Hound will give you